WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -1153,6 +1153,11 @@ private void setProgressVisible(boolean visible) {
     }
 
     private void onCommentModerated(OnCommentChanged event) {
+        // send signal for listeners to perform any needed updates
+        if (mNote != null) {
```

Comment:
Should we check if it was an error before firing a new event? (that in order does a new network call?)

Reply:
No sorry, better to keep it regardless of the status, since Notes list is updated/refreshed as soon as the user taps on the moderation/like action by using `onActivityResult`. In case of network errors we need to restore the old status.

Reply:
Exactly!  👍